release/models/bgp/openconfig-bgp-common-multiprotocol.yang

```diff
@@ -228,8 +237,10 @@ submodule openconfig-bgp-common-multiprotocol {
 
     leaf-list send-community-type {
       type oc-bgp-types:community-type;
+      status deprecated;
       description
-        "Specify which types of community should be sent to the
+        "This leaf has been deprecated because it was added by mistake.
+        Specify which types of community should be sent to the
```

Comment:
@missaesasaya it seems this is a logical and valid option.  From a consistency point of view, it is desirable to generate configuration using AFI/SAFI at each level (global/peer-group/neighbor).  

Can we keep this leaf?

Reply:
We currently have this leaf for neighbors/peers. If we keep this for global afi/safi, won't there be a conflict if we have different values? We would have two configurations that have no hierarchical relationship. It is different with the peer+afi/safi configuration where it is clear that the more specific configuration has precedence.